pkg/volume/secret/secret.go

```diff
@@ -45,8 +45,10 @@ type secretPlugin struct {
 
 var _ volume.VolumePlugin = &secretPlugin{}
 
-var wrappedVolumeSpec = volume.Spec{
-	Volume: &api.Volume{VolumeSource: api.VolumeSource{EmptyDir: &api.EmptyDirVolumeSource{Medium: api.StorageMediumMemory}}},
+func wrappedVolumeSpec() volume.Spec {
```

Comment:
I think it's ok/expected for util packages to live at the same level as the package that consumes them. So as long it doesn't result in circular dependencies that would be a good cleanup.

Regardless, any cleanup/refactoring can be done in a follow up.